src/askscript/__tests__/03-lambda/program07b-chain_lambda.ask

```diff
@@ -1,5 +1,4 @@
 ask {
-  let useFor = fun(x, f) { f(x) }
```

Comment:
Okay so this definition is much simpler than [src/askvm/resources/core/useFor.ts](https://github.com/xFAANG/askql/pull/232/files#diff-ae653ac9c4d4704ef382af488f3b64f6). Why are we adding a new resource implemented in TypeScript instead of a new standard function implemented in AskScript?

Reply:
We discussed it [10 days ago](https://github.com/xFAANG/askql/issues/43#issuecomment-644754817) and I implemented it according to your comments.